docs/get_started/getting_started_linux_cmake.md

```diff
@@ -107,6 +107,39 @@ $ ./build/iree/tools/iree-run-mlir $PWD/iree/tools/test/simple.mlir \
     -input-value="i32=-2" -iree-hal-target-backends=vmla -print-mlir
 ```
 
+### Dylib LLVM AOT backend
+
+To compile IREE LLVM AOT module we need to set the AOT linker path environment variable (eg: for lld-10):
+
+```shell
+$ export IREE_LLVMAOT_LINKER_PATH=ld.lld-10
+```
+
+Translate a source MLIR into an IREE module:
+
+```shell
+# Assuming in IREE source root
+$ ./build/iree/tools/iree-translate \
+    -iree-mlir-to-vm-bytecode-module \
+    -iree-llvm-target-triple=x86_64-linux-gnu \
+    -iree-hal-target-backends=dylib-llvm-aot \
+    iree/tools/test/simple.mlir \
+    -o /tmp/simple-llvm_aot.vmfb
+```
+
+Then push the IREE runtime executable and module to the device:
```

Comment:
This line from Android steps makes less sense when running the code on the host device (Linux).

How about this?

```suggestion
Then run the compiled module using the `dylib` HAL driver:
```